content/en/observability_pipelines/log_enrichment/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Log Enrichment
+disable_toc: false
+---
+
+## Overview
+
+Use the Log Enrichment Template to enrich and transform your logs with Observability Pipelines's processors before sending them to its destination.
+
+{{% observability_pipelines/use_case_images/log_enrichment %}}
+
+Select a source to get started:
+
+- [Datadog Agent][1]
+- [Fluentd or Fluent Bit][2]
+- [HTTP Client][3]
+- [Splunk HTTP Event Collector (HEC)][4]
+- [Splunk Heavy and Universal Forwarders (TCP)][5]
+- [Sumo Logic Hosted Collector][6]
+- [Rsyslog or Syslog-ng][7]
+
+[1]: /observability_pipelines/dual_ship_logs/datadog_agent
+[2]: /observability_pipelines/dual_ship_logs/fluent
+[3]: /observability_pipelines/dual_ship_logs/http_client
+[4]: /observability_pipelines/dual_ship_logs/splunk_hec
+[5]: /observability_pipelines/dual_ship_logs/splunk_tcp
+[6]: /observability_pipelines/dual_ship_logs/sumo_logic_hosted_collector
+[7]: /observability_pipelines/dual_ship_logs/syslog
```

Comment:
```suggestion
[1]: /observability_pipelines/log_enrichment/datadog_agent
[2]: /observability_pipelines/log_enrichment/fluent
[3]: /observability_pipelines/log_enrichment/http_client
[4]: /observability_pipelines/log_enrichment/splunk_hec
[5]: /observability_pipelines/log_enrichment/splunk_tcp
[6]: /observability_pipelines/log_enrichment/sumo_logic_hosted_collector
[7]: /observability_pipelines/log_enrichment/syslog
```

Reply:
Thanks, good catch! I'm going to add a bit more context to the overviews so might need a review of that as well.